src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -127,10 +127,10 @@ public static List<Person> getTypicalPersonsDescendingName() {
     }
 
     public static List<Person> getTypicalPersonsAscendingDateOfLastVisit() {
-        return new ArrayList<>(Arrays.asList(ALICE, DANIEL, BENSON, GEORGE, FIONA, ELLE, CARL));
+        return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
```

Comment:
Just a note on this now the ascending date of last visit is also the ascending name order. This is inadvisable because we may get false positives while testing where we sort by ascending name and we think we have sorted by ascending date of last visit. Hence, I would recommend at least keeping one name out of order.